### DISCUSSION TITLE
Can't view who starred or forked a repo anymore

### DISCUSSION THREAD

**Discussion by galaxyhaxz** [community/community#8671] [Projects and Issues] [answered]
New design is inconvenient and took away the ability to click on the buttons and see who has starred the repo.

**Comment by philipturner** [comment]
I agree. GitHub staff, please let us use the old UI again.

**Reply by tinyhaggis** [reply]
It has returned!

Edit:  Well, not really.  You still can't click on the fork count to see the forks.

**Reply by tomhundt** [reply]
They could add to the dropdown for Fork and Star a "View Others'" option -- having the same effect as clicking the relevant item under About, on the right.  These buttons/dropdowns are current-user centered (i.e., I could fork something 3 times and they would be listed on this dropdown under Fork).  Confusing if you don't realize this is how it's oriented.  They probably couldn't think of any better place to put that functionality.

**Comment by leahein** [comment, marked as answer]
That information is actually still available, it's moved to the right side, in the **About** section.
![image](https://user-images.githubusercontent.com/15165051/145655054-cb7907c3-b27a-4ec8-8e0a-3e5556592605.png)

**Reply by galaxyhaxz** [reply]
Just checked a few repos and it seems you're right. I'm not sure if they just implemented this or I didn't notice it in the first place.

Will take a little bit of getting used to been used to the old style for 7 years lol.

Edit: one of the downsides to the new design though is you must be on the front page of the repo for that to appear. If you click on a folder or file it removes the side bar.

**Reply by philipturner** [reply]
The solution works for me

**Reply by Boruch-Baum** [reply]
That entire section of the front-page is redundant, and the duplication can only cause confusion. I vote it as awful UX. I don't care where the features (to act and to view) are, but they should be at a single point on the page. And (grumble) (off-topic) the list of forks should include data like how far ahead or behind each is.

**Reply by tinyhaggis** [reply]
Why do big developers always make the same assumption that obfuscation of features through more clicks is a good thing?  I should be able to view the forks for a repo from any point in that repo with a single click, just like I used to be able to.

**Comment by philipturner** [comment]
Your comment about forks would be especially helpful. I think we should have an option to sort forks by how up-to-date they are as opposed to by name. This could realistically happen if it is a feature turned off by default.

**Comment by Boruch-Baum** [comment]
A year (few years?) ago, I requested a version of the feature to some
github tech support and got a non-commital reply.

On 2022-01-04 19:07, Philip Turner wrote:
>    Your comment about forks would be especially helpful. I think we should
>    have an option to sort forks by how up-to-date they are as opposed to
>    by name. This could realistically happen if it is a feature turned off
>    by default.

--
hkp://keys.gnupg.net
CA45 09B5 5351 7C11 A9D1  7286 0036 9E45 1595 8BC0

**Reply by Pororo1st** [reply]
Sad

**Comment by philipturner** [comment]
Typical of big companies. I’ve had the same instance of being ignored by Apple (see end of README at https://github.com/philipturner/metal-fft) and expect it from Google as I attempt to resurrect Swift for TensorFlow. Good luck on any of your efforts!

**Comment by tinyhaggis** [comment]
Why was this confusing change made?  Now we need to be on the front page of a repo to see its forks, or we have to dig through the Network section inside Insights to see them.  Terrible decision.

**Comment by domsson** [comment]
While I have to admit that I often accidentally forked or starred a repository when I just wanted to see who had already done so, I also agree that having this information moved to the sidebar is inconvenient; mainly for the reason already mentioned: only visible on the main page of a repo. I wasn't able to find the functionality until I came here.

I would prefer if we could keep the functionality where it originally was and make it less confusing in terms of how to star/ fork vs. how to see stargazers/ forkers.

**Reply by philipturner** [reply]
What about if there was an option in user settings where you can re-enable the old behavior?

**Reply by tinyhaggis** [reply]
@philipturner That would be the best approach.  The old functionality just worked.  I must have created 4-5 forks because of muscle memory when checking an interesting repo before I retrained my brain to go to the front page of the repo first.

**Comment by tinyhaggis** [comment]
I see the Forks dropdown has reappeared as of a couple of days ago.  Thanks for adding it back in, GitHub!  It still doesn't let you click on the number of forks to see all forks for a repo though, so some work still needed.

**Comment by quantuminformation** [comment]
i used to love seeing who starred by repos

**Comment by yashu1wwww** [comment]
Github Tools:
https://yashawanthtechprojects.on.drv.tw/GitHub/uploaddatefinder.html

**Reply by tomhundt** [reply]
404 on that link

**Comment by samjco** [comment]
It is important to see who all forked a repo... because if the original owner stops supporting a particular project, how are we going to know if some else carry on the project??
